src/data/http/http.ts

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import { refreshToken as refreshTokenService } from '../auth/authServices';
+
+const axiosInstance = axios.create({
+  baseURL: 'http://173.212.214.70:3001/auth/',
```

Comment:
The baseUrl should be `http://173.212.214.70:3001`. And won't forget to add `/auth/` segment to the pathes of the auth services